src/devices/bus/coco/coco_ssc.cpp

```diff
@@ -102,6 +103,7 @@ namespace
 		uint8_t                                 m_tms7000_portb;
 		uint8_t                                 m_tms7000_portc;
 		uint8_t                                 m_tms7000_portd;
+		emu_timer                               *m_timer;
```

Comment:
Please give this a more descriptive name - `m_timer` doesn't really say what it times.